### DISCUSSION TITLE
zh/ch3

### DISCUSSION THREAD

**Discussion by giscus** [js-regex-mini-book/js-regex-mini-book.github.io#6] [Announcements]
# zh/ch3

JavaScript 正则表达式迷你书

https://js-regex-mini-book.github.io/zh/ch3.html

**Comment by zhh1137152324** [comment]
return str.replace(/^\s*(.*?)\s\*$/g, "$1"); 改成
return str.replace(/^\s*(.*?)\s*$/g, "$1");
